### PR TITLE
Refactored the PTQ test

### PR DESCRIPTION
**Pull request by AlexKoff88**
- Refactored the code for the sake of extensibility
- Limit the validation dataset size and model scope to reduce the test time and tun it on nightly basis

### CODE REVIEW

File: tests/post_training/model_scope.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 def get_validation_scope() -> List[dict]:
     model_scope = []
     # Basic
-    model_scope.append({"name": "vgg11", "quantization_params": {}})
+    #model_scope.append({"name": "vgg11", "quantization_params": {}})
```
Comment (AlexKoff88) [inline tests/post_training/model_scope.py:9]: Commenting the code is not good. But I would like to keep these models to get back in the future and for ramping up quickly when doing local experiments.

File: tests/post_training/test_quantize_conformance.py [hunk 2]
```diff
@@ -1,100 +1,181 @@
...
-import timm
-from tqdm import tqdm
+from model_scope import get_validation_scope
 from sklearn.metrics import accuracy_score
+from utils import (create_timm_model, export_to_ir, export_to_onnx,
+                   get_model_transform, get_torch_dataloader, run_benchmark)
 
-import openvino.runtime as ov
+import nncf
+from nncf.common.utils.logger import logger as nncf_logger
```
Comment (alexsu52) [inline tests/post_training/test_quantize_conformance.py:20]: Build 3 was failed with following error:
[2022-12-20T12:14:21.429Z] E   ModuleNotFoundError: No module named 'nncf.common.utils.logger'

Please, fix import.

**Comment by alexsu52** [comment]
Run PTQ: job 3

File: tests/post_training/model_scope.py [hunk 3]
```diff
@@ -18,30 +18,30 @@ def get_validation_scope() -> List[dict]:
...
     model_scope.append({"name": "hrnet_w18", "quantization_params": {"preset":QuantizationPreset.MIXED}})
-    model_scope.append({"name": "efficientnet_lite0", "quantization_params": {"preset":QuantizationPreset.MIXED}})
-    model_scope.append({"name": "ghostnet_050", "quantization_params": {"preset":QuantizationPreset.MIXED}}) 
+    # model_scope.append({"name": "efficientnet_lite0", "quantization_params": {"preset":QuantizationPreset.MIXED}})
+    # model_scope.append({"name": "ghostnet_050", "quantization_params": {"preset":QuantizationPreset.MIXED}}) 
     model_scope.append({"name": "dpn68", "quantization_params": {"preset":QuantizationPreset.MIXED}})
-    model_scope.append({"name": "dla34", "quantization_params": {"preset":QuantizationPreset.MIXED}})
+    # model_scope.append({"name": "dla34", "quantization_params": {"preset":QuantizationPreset.MIXED}})
     
     return model_scope
```
Comment (KodiaqQ) [inline tests/post_training/model_scope.py:47]: Newline missed.

File: tests/post_training/test_quantize_conformance.py [hunk 4]
```diff
@@ -1,100 +1,181 @@
...
+class QuantizerBase(metaclass=ABCMeta):
+    """Base class for an executor"""
 
-    return transform
+    def __init__(self, name):
+        super().__init__()
+        self.name = name
 
+    @abstractmethod
+    def quantize(
```
Comment (KodiaqQ) [inline tests/post_training/test_quantize_conformance.py:38]: It would be nice to have a little fix for such code roughnesses if that's possible of course.

**Comment by daniil-lyakhov** [comment]
@AlexKoff88 I did something similar to this changes in my PR recently https://github.com/openvinotoolkit/nncf/pull/1591
Please rebase your changes

**Pull request closed without merging**